xmppserver/src/main/webapp/session-details.jsp

```diff
@@ -359,14 +359,20 @@
             } %>
         </td>
     </tr>
-    <% 
+</tbody>
+</table>
+<table cellpadding="0" cellspacing="0" border="0" width="100%">
```

Comment:
I'd only show this table when getSoftwareVersion != null.

Reply:
I already updated it on https://github.com/igniterealtime/Openfire/pull/1379/commits/76f25c6dc41d4730be9b554717b2080b5421b62a

```